### DISCUSSION TITLE
Problem with wpa_supplicant.conf file

### DISCUSSION THREAD

**Discussion by leovernerey** [awawa-dev/HyperHDR#763] [General]
Hello,

I made an ambilight system using a raspberry pi 0 wifi and an esp8266 with HyperSPI. When I created the wpa_supplicant.conf on the RPI0 and booted the complet system it worked well, but when I rebooted it (disconnect/connect the power supply), it didn't work. I find that if I recreate again the supplicant file it works again. Do you know how can I fix that problem ?

**Comment by awawa-dev** [comment]
Hi
wpa_supplican.conf is no longer supported in latest official Raspberry Pi Bookworm release. Read more: https://github.com/awawa-dev/HyperHDR/issues/699
If you are using an older edition of Bullseye it should work, otherwise there is something wrong with the wpa_supplican.conf itself, especially this file does not like Windows "new line chars".
